plugins/cost-insights/src/components/CostOverviewCard/CostOverviewCard.tsx

```diff
@@ -60,22 +59,53 @@ const CostOverviewCard = ({
           <PeriodSelect duration={filters.duration} onSelect={setDuration} />
         </CostOverviewHeader>
         <Divider />
-        <Box marginY={1} display="flex" flexDirection="column">
-          <CostOverviewChartLegend change={change} title={`${name} Trend`} />
+        <Box my={1} display="flex" flexDirection="column">
+          <Box display="flex" flexDirection="row">
+            <Box mr={2}>
+              <LegendItem
+                title={`${dailyCost.name} Trend`}
+                markerColor={theme.palette.blue}
+              >
+                {formatPercent(data[0].change.ratio)}
+              </LegendItem>
+            </Box>
+            {metric && metric.kind && data[1] && comparedChange && (
+              <>
+                <Box mr={2}>
+                  <LegendItem
+                    title={metric.name}
```

Comment:
Should be 
```
title={`${metric.name} Trend`}
```
 to match above

Reply:
Can do but we don't do this internally.

Reply:
True - I think we should. I added "Trend" above recently since "Daily Cost: 6%" made no sense, the same applies for the metrics imo.